dtensor-keras-bert.py

```diff
@@ -27,54 +27,62 @@
 
```

Comment:
Might want to port this to my experimental folder as well.

Reply:
Yes we shall avoid the divergence. I'll file a CL. I think it is mostly a direct copy of the file.